### PR TITLE
 fix : broken traffic cam

### PR DESCRIPTION
**Pull request by sandipa12**
The url for Tobey street was broken so it has been replaced with the new url.  https://www.pivotaltracker.com/story/show/178115637

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -127,6 +127,6 @@
 TrafficCam.find_or_initialize_by(
   title: "Tobey Street"
```
Comment (cbarber) [inline db/seeds.rb:128]: I think the title should probably change as well.

**Comment by tucker-weed** [comment]
A part of the URL changed from Tobey to Dean - is the new URL for a different street then?

**Review by cbarber — COMMENTED** [review]
This is a great start. I think we should probably add a migration to manipulate existing data in databases:
https://guides.rubyonrails.org/v5.2/active_record_migrations.html

**Comment by sandipa12** [comment]
@tucker-weed  Yes, the new url is for a different state (Dean Street)

File: db/seeds.rb [hunk 2]
```diff
@@ -125,8 +125,8 @@
 end.save!
 
 TrafficCam.find_or_initialize_by(
-  title: "Tobey Street"
+  title: "Dean Street"
 ) do |r|
-  r.url = "http://www.dot.ri.gov/img/travel/camimages/6-2%20Rt%206%20E%20%20%2010%20N%20@%20Tobey%20St.jpg"
+  r.url = "https://www.dot.ri.gov/img/travel/camimages/6-1%20Rt%206%20E%20%20%2010%20N%20@%20Dean%20St.jpg"
```
Comment (tmsavage) [inline db/seeds.rb:130]: What would be the purpose of changing http to https for the link?

Comment (sandipa12) [reply db/seeds.rb:130]: It does not really matter. So far I know, https is more secure than http but either of them works to open the link.

**Comment by chewnoill** [comment]
We can squash these commits together, but other wise LGTM

File: db/seeds.rb [hunk 3]
```diff
@@ -133,8 +133,8 @@
...
```
Comment (fallynlogan) [inline db/seeds.rb:138]: Would the logic & where you retrieved the street cam info be similar for adding traffic cams for other locations?

Comment (sandipa12) [reply db/seeds.rb:138]: @fallynlogan Yeah, it  should be the same.

File: db/seeds.rb (hunk 2 above)
Comment (bencates) [reply db/seeds.rb:138]: You should basically always prefer `https` to `http` when you have the choice, even if you don't directly care about any of the security features that TLS gives you.

Pages served to the user via HTTP will happily download HTTPS assets. However, pages served over HTTPS will refuse to fetch HTTP assets as doing so could compromise the security guarantees HTTPS makes for the page as a whole.

**Review by cbarber — CHANGES_REQUESTED** [review]
I think we should use `rake db:seed` to update production until we have an admin interface to change these values. In this case, we'll need our seeds.rb to delete the old `Tobey Street` camera in addition to adding a new `Dead Street`.

Any objections to adding?
`TrafficCam.delete_by(title: "Tobey Street")`

**Review by chewnoill — DISMISSED** [review]
Adding a delete seems like a good idea for the old row. As long as it can be run multiple times without breaking anything. I'm also okay with dropping the database and reseeding if we run into any problems.

**Review by chewnoill — APPROVED** [review]

**Review by cbarber — APPROVED** [review]
Looks like the API is `destroy_all` or `delete_all`. `destroy_all` triggers ActiveRecord callbacks for the destroy event, and `delete_all` sends the SQL directly without any callbacks. Nice work. :+1:

**Pull request merged**